packages/swapper/src/swappers/zrx/getZrxTradeQuote/getZrxTradeQuote.ts

```diff
@@ -32,22 +36,26 @@ export async function getZrxTradeQuote(input: GetTradeQuoteInput): Promise<Trade
     const useSellAmount = !!sellAmount
     const buyToken = buyAssetNamespace === 'erc20' ? buyAssetErc20Address : buyAsset.symbol
     const sellToken = sellAssetNamespace === 'erc20' ? sellAssetErc20Address : sellAsset.symbol
-
     const { minimum, maximum } = await getZrxMinMax(sellAsset, buyAsset)
-
     const minQuoteSellAmount = bnOrZero(minimum).times(bn(10).exponentiatedBy(sellAsset.precision))
 
     const normalizedSellAmount = normalizeAmount(
       bnOrZero(sellAmount).eq(0) ? minQuoteSellAmount : sellAmount
     )
+    const baseUrl = baseUrlFromChainId(buyAsset.chainId)
+    if (!baseUrl)
```

Comment:
might make our lives easier if the `baseUrlFromChainId` throws the `SwapError` so we don't have to check for undefined and throw everywhere